saleor/graphql/tax/tests/mutations/test_tax_class_update.py

```diff
@@ -141,3 +141,33 @@ def test_remove_country_rates(staff_api_client, permission_manage_taxes):
     data = content["data"]["taxClassUpdate"]
     assert not data["errors"]
     assert len(data["taxClass"]["countries"]) == 0
+
+
+def test_remove_individual_country_rates(staff_api_client, permission_manage_taxes):
```

Comment:
Could you add a test about removing not existing rate?

Reply:
Added